pkg/redis/RedisConsumer.php

```diff
@@ -21,12 +21,42 @@ class RedisConsumer implements Consumer
      */
     private $context;
 
+    /**
+     * @var int
+     */
+    private $retryDelay;
+
+    /**
+     * @var RedisQueueConsumer
+     */
+    private $queueConsumer;
+
     public function __construct(RedisContext $context, RedisDestination $queue)
     {
         $this->context = $context;
         $this->queue = $queue;
     }
 
+    /**
+     * @return int
+     */
+    public function getRetryDelay(): ?int
+    {
+        return $this->retryDelay;
+    }
+
+    /**
+     * @param int $retryDelay
+     */
+    public function setRetryDelay(int $retryDelay): void
+    {
+        $this->retryDelay = $retryDelay;
+
+        if ($this->queueConsumer) {
```

Comment:
I think setting retryDelay property is enough. No need to set same value on queue consumer too. (It will be passed as argument to receiveXXX methods).